Lib/shlex.py

```diff
@@ -7,11 +7,7 @@
 # iterator interface by Gustavo Niemeyer, April 2003.
 # changes to tokenize more like Posix shells by Vinay Sajip, July 2016.
 
-import os
```

Comment:
Could you add tests for these using `ensure_lazy_imports()`?

Reply:
Good idea, done.